www/content/staff/curt-newton.md

```diff
@@ -1,7 +1,7 @@
 ---
 first_name: Curt
 last_name: Newton
-job_title: OCW Publication Director
```

Comment:
> change Curt Newton's title from "Publication Director" to simply "Director"

Currently it's `OCW Publication Director` so should it be changed to only `Director` or `OCW Director`? 
@pdpinch

Reply:
Good question. I don't know why some titles have "OCW" in front of them, when most do it. Let's remove it.